Automattic/wp-calypso
# Site migration: progress view adjustments

Note: This change is behind a feature flag and is part of an in-progress larger project. For context, see pbkcP4-8-p2.

#### Changes proposed in this Pull Request

* Replaces illustration.
* Wraps domains in subheader in spans to prevent them breaking across lines.
* Sizes and aligns progress item icons.

![image](https://user-images.githubusercontent.com/1647564/70260353-ffed7f80-1787-11ea-8555-be0cb4a350e6.png)

#### Testing instructions



* Check out this pull request and run Calypso locally, or view it on calypso.live with the flag tools/migrate.
* Viewing calypso.localhost:3000/migrate/, you should see a SiteSelector component allowing you to choose a site to migrate a Jetpack site into.
* When you select a site, you should see another SiteSelector component, listing all the Jetpack sites your account has access to.
* Selecting one of those sites should take you to a confirmation screen.
* Accept the confirmation. You should see a progress screen detailing the steps of the migration.
  * The illustration should not have a coloured bubble.
  * The progress list item icons should be aligned.
  * The domains in the subheader should not break when you resize your viewport.
* When the migration is done you should see a message telling you it is complete.
